README.md

```diff
@@ -8,9 +8,15 @@ This project has several scripts for different automated tasks, some of them are
 
 * create-vm.sh, creates a fresh KVM VM via libvirt.
 * crowbar-prep.sh, prepare a host for Crowbar admin node installation.
-* [mkcloud.sh](docs/mkcloud.md), script used to build a SUSE Cloud environment
+* [mkcloud](docs/mkcloud.md), script used to build a SUSE Cloud environment
```

Comment:
Better to surround script names with backticks, and maybe also include the relative path:

    [`scripts/mkcloud`](docs/mkcloud.md)